css/src/components/form/textarea.scss

```diff
@@ -87,3 +87,60 @@ $textarea-focus-box-shadow-size: 0 0.0625rem 0 0 !default;
 		}
 	}
 }
+
+$textarea-form-gap: 0.25em !default;
+$textarea-divider-radius: 0.5rem !default;
+
+.textarea-form {
+	display: grid;
+	gap: $textarea-form-gap;
+	padding-block: $control-padding-vertical;
+	padding-inline: $control-padding-horizontal;
+	border: $control-border-width solid transparent;
+	border-radius: $border-radius-lg;
+	border-color: $textarea-border-color;
+	background-color: $textarea-background-color;
+	font-size: $control-font-size;
+	border-block-end-color: $textarea-border-bottom-color;
+	outline-color: transparent;
+	transition: box-shadow 300ms linear;
+
+	@include prefers-reduced-motion {
```

Comment:
I like this transition, should we apply it to all of the input boxes like input, textarea, and select? It's not the same animation as fluent 2 though :/

Reply:
Done!